portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -12,17 +15,17 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  public ArrayList<String> com;
+  public ArrayList<String> com = new ArrayList<String>();
 
-  @Override
-  public void init() {
-    com = new ArrayList<String>(); 
-    com.add(
-        "Those who think they can change the world are the ones who do. - ");
-    com.add("They told me computers could only do arithmetic. - ");
-    com.add("A ship in port is safe, but thats not what ships are built for. - ");
-    // console.log("ArrayList : " + com); 
-  }
+//   @Override
+//   public void init() {
+//     com = new ArrayList<String>(); 
+//     com.add(
+//         "Those who think they can change the world are the ones who do. - ");
+//     com.add("They told me computers could only do arithmetic. - ");
+//     com.add("A ship in port is safe, but thats not what ships are built for. - ");
+//     console.log("ArrayList : " + com); 
+//   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
doGet() still reads comments from 'com' ArrayList. I think you need to read comments from Datastore here, otherwise each servlet restart will clear comments.

Reply:
:) I posted comment at the same time when you posted this implementation